### PR TITLE
Add bidder and switch for Ozone, update Prebid dependency

### PR DESCRIPTION
**Pull request by katebee**
#### 👉  Add a switch and client-side bidder for Ozone

First steps to using an adapter rather than the server-side component.

#### 👉Update Prebid to version 2.6.0 with some extra modules

Latest Prebid release includes required updates to the ozone adapter, the build referenced also includes two required modules.

See PR: https://github.com/guardian/Prebid.js/pull/91 

## Screenshots

Screenshot 2019-03-13 at 17 27 08

## What is the value of this and can you measure success?

Commercial BAU: https://trello.com/c/eOnbioKJ

Bidder is behind a switch, however we can use the test params to test this on PROD.

Currently using test params but this is required until we get the adapter working correctly.

### Tested

- [x] Locally

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/prebid/prebid.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ type S2SConfig = {
     bidders: Array<string>,
     timeout: number,
     adapter: string,
-    is_debug: string, // true or false string
+    is_debug: 'true' | 'false',
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/commercial/modules/prebid/prebid.js]: Nice!

File: static/src/javascripts/projects/commercial/modules/prebid/utils.js [hunk 2]
```diff
@@ -113,6 +113,10 @@ export const shouldIncludeAdYouLike = (slotSizes: PrebidSize[]): boolean =>
 export const shouldIncludeOzone = (): boolean =>
     !isInUsRegion() && !isInAuRegion();
 
+// TODO: Check is we want regional restrictions on where we load the ozoneBidAdapter
+export const shouldUseOzoneAdaptor = (): boolean =>
+    !isInUsRegion() && !isInAuRegion() && config.get('switches.prebidOzone');
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/commercial/modules/prebid/utils.js]: I think the switch test is done automatically, which is why you add the switchName when you set up the bidder at https://github.com/guardian/frontend/pull/21251/files#diff-3504a8960b869185b559c4491c6a5857R372

**Comment by jeteve** [comment]
Hi! How does that play with Analytics? Does every event turn up as just 'ozone' with this solution? I reckon we need to solve that to leave the analytics as they are (maybe we can ingest separately?), unless our D&I friend state otherwise.

**Comment by katebee** [comment]
@jeteve plan is to get this on PROD but leave it switched off 😄 

The events are currently turning up as `ozone` which does not bode well for the analytics. However, I am uncertain if this is an artefact of the test IDs, configuration within ozone, or running it locally.

Merging this as-is to PROD will allow me to further troubleshoot using the pbtest params, and allow external folks to also help us get this working correctly.

File: static/src/javascripts/projects/commercial/modules/prebid/utils.js (hunk 2 above)
Comment (katebee) [reply static/src/javascripts/projects/commercial/modules/prebid/utils.js]: Good point! I will remove it in the next iteration. 

Plan is to get this on PROD but keep it switched off so I can do some testing with the pbtest params, and better communicate the issue externally.

**Pull request merged**